lib/etcdctl.py

```diff
@@ -178,14 +177,20 @@ def run(self, arguments, endpoints: Optional[str] = None, api=3):
             command.insert(2, endpoints)
 
         try:
-            return check_output(command, env=env, stderr=STDOUT).decode("utf-8")
+            proc = run(command, env=env, check=True, capture_output=True)
```

Comment:
We need to decode both stdout and stderr in the except clause, right?

Reply:
Done